src/plugins/ConsolidatePlugin.ts

```diff
@@ -22,6 +26,8 @@ export class ConsolidatePluginClass implements Plugin {
     this.engine = options.engine;
     this.extension = options.extension || `.${options.engine}`;
     this.useDefault = (options.useDefault !== undefined) ? options.useDefault : true;
+    this.basedir = (options.basedir !== undefined) ? options.basedir : file.context.homeDir;
```

Comment:
file variable does not exist

Reply:
@nchanged said that it was...
so idk.

Reply:
You can't initialize it in the constructor. It has to be done on each file